src/entities.ts

```diff
@@ -187,4 +188,56 @@ customEvents.on('gameLoaded', () => {
     }
 
   })
+
+  const updateAllEntitiesInTeams = () => {
+    for (const entity of Object.values(bot.entities)) {
+      if (entity.type === 'player' && entity.username && bot.teamMap[entity.username] || entity.uuid && bot.teamMap[entity.uuid]) {
+        bot.emit('entityUpdate', entity)
+      }
+    }
+  }
+
+  bot.on('teamUpdated', (team: Team) => {
+    if (appViewer.playerState.reactive.team?.team === team.team) {
+      // Player team was updated, need to update all entities that are in a team
```

Comment:
That comment is kind of confusing, why do we need to update absolutely all entities? (Sorry maybe the reason is obvious and it's fine but still can't get it). How often it might happen (in what server events)? just want to make sure we can skip optimising this if it's really rare